content/kit/wiring.md

```diff
@@ -1,3 +1,14 @@
 ---
 title: Wiring
 ---
+
+Ensuring your robot is wired correctly is incredibly important. If wired incorrectly, your kit may not work correctly, or may be damaged.
+
+## Wire Colouring
+Ground should be black, and nothing else should be black. This is in the interest of both safety and organisation. Being able to see where all your ground cables go allows you to easily track cable runs and diagnose electronic issues. It also greatly reduces the chance of running an actual current down the ground, which will seriously damage the kit!
+
+## Keep your solders clean
+Sometimes, you'll need to build your own circuits to extend out kit, which likely requires soldering. Keep an eye on your solders so they don't create a bridge between components. This will likely lead to the electronics not working correctly.
```

Comment:
First sentence: "extend out kit" -> "extend our kit" ?

I'd also recommend changing "solders" to "solder joints". Maybe its just because I'm really tired but wasn't sure exactly what you meant by "solders" until I'd read it a few times.